bundles/org.openhab.binding.danfossairunit/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="danfossairunit"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!--HRV -->
+	<thing-type id="airunit">
+		<label>Danfoss Air Unit</label>
+		<description>The Danfoss Air Unit Heat Exchanger, CCM and Air Dial</description>
+
+		<channel-groups>
+			<channel-group id="main" typeId="main" />
+			<channel-group id="temps" typeId="temps" />
+			<channel-group id="humidity" typeId="humidity" />
+			<channel-group id="recuperator" typeId="recuperator" />
+			<channel-group id="service" typeId="service" />
+		</channel-groups>
+		<properties>
+			<property name="Unit Name">unknown</property>
+			<property name="Serial Number">unknown</property>
+		</properties>
```

Comment:
Are these properties meant to get populated elsewhere?

Reply:
These are the fallbacks/defaults which are overwritten with values received from the air unit, once it is connected during initialization.